posts.yml

```diff
@@ -269,7 +338,7 @@ posts:
   - title: Xmas party 2018 (CocoaHeads no. 121)
     subtitle: CocoaHeads, Android GDG & Melbourne Mobile
     created_at: 2018-12-08 00:00:00 +0
-    frontpage: 3
+    frontpage: 0
```

Comment:
Is that what we want?
Is frontpage order `0` the first item to be presented?

Reply:
The logic for front-page is > 0 :P